test/units/testsuite-58.sh

```diff
@@ -1,71 +1,87 @@
 #!/usr/bin/env bash
 set -eux
+set -o pipefail
+
+if ! which systemd-repart &>/dev/null; then
```

Comment:
As @bluca said, please use `command -v` instead. Also, right now this attempts to skip the test on Arch because of:

```
bash-5.0# which systemd-repart
bash: which: command not found
```

so just in case I'd redirect only stdout.